packages/indexer-common/src/types.ts

```diff
@@ -90,3 +95,14 @@ export interface TransactionConfig extends providers.TransactionRequest {
   gasBump: BigNumber
   type: TransactionType
 }
+
+export function parseDeploymentManagementMode(input: string): DeploymentManagementMode {
```

Comment:
For allocation management mode we do something like this to parse the input to the enum variant value: 

```
const allocationManagementMode =
      AllocationManagementMode[
        argv.allocationManagement.toUpperCase() as keyof typeof AllocationManagementMode
      ]
```     
This approach may be a bit more flexible to support adding enum variants without needing to change the parsing function.  What do you think?